manim/mobject/matrix.py

```diff
@@ -81,9 +107,36 @@ def __init__(
         **kwargs,
     ):
         """
-        Matrix can either either include numbers, tex_strings,
-        or mobjects
+
+        Parameters
+        ----------
+        matrix : np.array|List[List]
+            A numpy 2d array or list of lists
+        v_buff : float, optional
+            vertical buffer, by default 0.8
+        h_buff : float, optional
+            horizontal buffer, by default 1.3
+        bracket_h_buff : float, optional
+            bracket horizonal buffer, by default MED_SMALL_BUFF
+        bracket_v_buff : [type], optional
```

Comment:
Question: What's type?

Reply:
thanks for catching it. There are 2 more locations where I did not fill the type. Fixing it now.